test/integration/etcd_storage_path_test.go

```diff
@@ -677,20 +729,73 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	// --
 }
 
-// Be very careful when whitelisting an object as ephemeral.
+// Be very careful when whitelisting a kind as unexposed.
 // Doing so removes the safety we gain from this test by skipping that object.
-var ephemeralWhiteList = createEphemeralWhiteList(
-	// github.com/openshift/origin/pkg/authorization/apis/authorization/v1
+// These types aren't exposed via the API
+var unexposedGVKWhiteList = createUnexposedWhiteList(
+
+	gvk("", "v1", "RangeAllocation"),     // stored in various places in etcd but cannot be directly created // TODO maybe possible in kube
+	gvk("", "v1", "SerializedReference"), // used for serilization, not stored in etcd
+	gvk("", "v1", "NodeConfigSource"),    // subfield of node.spec, but shouldn't be directly created
+	gvk("", "v1", "PodStatusResult"),     // wrapper object not stored in etcd
+
+	// TODO this one appears to be broken
+	gvk("admissionregistration.k8s.io", "v1alpha1", "InitializerConfiguration"), // not stored in etcd
+
+	gvk("admission.k8s.io", "v1beta1", "AdmissionReview"), // not stored in etcd
+
+	gvk("autoscaling", "v1", "Scale"), // not stored in etcd
+
+	gvk("batch", "v1beta1", "JobTemplate"),  // not stored in etcd
+	gvk("batch", "v2alpha1", "JobTemplate"), // not stored in etcd
+
+	// TODO validation is failing on fields that don't exist
```

Comment:
upstream issue open for this?

Reply:
> upstream issue open for this?

post rebase issue that I plan to follow up on.  Events are transient, so I wasn't overly worried for the very short term.